core/src/test/java/io/questdb/test/griffin/GroupByTest.java

```diff
@@ -1404,6 +1404,124 @@ public void testGroupByWithDuplicateSelectColumn() throws Exception {
         );
     }
 
+    @Test
+    public void testGroupByWithLeftJoin() throws Exception {
+        assertMemoryLeak(() -> {
+            ddl(
+                    "create table dim_apTemperature as (" +
+                            "  select x::int id," +
+                            "         rnd_str('a','b','c') as category," +
+                            "         rnd_float() aparent_temperature" +
+                            "  from long_sequence(10)" +
+                            ");"
+            );
+            ddl(
+                    "create table fact_table as (" +
+                            "  select x::int id_aparent_temperature," +
+                            "         (x * 120000000)::timestamp date_time," +
+                            "         rnd_float() radiation," +
+                            "         rnd_float() energy_power" +
+                            "  from long_sequence(10)" +
+                            ");"
+            );
+
+            final String expectedResult = "dim_ap_temperature__category\tfact_table__date_time_day\tfact_table__avg_radiation\tfact_table__energy_power\n" +
```

Comment:
nit: we should use "assertSql" (it is simpler and faster than "assertQuery") and also assert execution plan.

Reply:
Done in bfa49e7c6fcdc9ba8a448a0726003cb51560e0ed